monitoring/mongodb/dashboard.py

```diff
@@ -486,84 +818,88 @@ def mongodb_state_timeline(title, expr, description="", mappings=None, **kwargs)
     ],
     targets=[
         Target(
-            expr=f'max by (job) (mongodb_mongos_sharding_chunks_is_balanced{{namespace="{NAMESPACE}"}})',
-            legendFormat="{{job}}",
+            expr="max by (job) (" + Metrics.MONGOS_SHARDING_CHUNKS_IS_BALANCED() + ")",
+            legendFormat="{{pod}}",
         )
     ],
 )
 
 mongos_cursors = mongodb_stat(
     "MongoS cursors open against selected instances",
     decimals=0,
-    targets=[Target(expr=MongoDBMetrics.cursors_open(), legendFormat="{{job}}")],
+    targets=[
+        Target(
+            expr="sum by (job) (" + Metrics.SS_METRICS_CURSOR_OPEN() + ")",
+            legendFormat="{{pod}}",
+        )
+    ],
 )
 
 active_connections = mongodb_timeseries(
     "Active connections",
     [
         Target(
-            expr=MongoDBMetrics.connections("current"),
+            expr="sum by(job) (" + Metrics.SS_CONNECTIONS(conn_type="current") + ")",
             legendFormat="__auto",
         )
     ],
     legendDisplayMode="table",
     legendPlacement="right",
     thresholds=THRESHOLDS["connections"],
-    fillOpacity=0,
     showPoints="auto",
 )
 
 active_cursors = mongodb_timeseries(
     "Active cursors",
     [
         Target(
-            expr=f'max(mongodb_ss_metrics_cursor_open{{job=~"$job", namespace="{NAMESPACE}"}})',
+            expr="max(" + Metrics.SS_METRICS_CURSOR_OPEN() + ")",
             legendFormat="Open cursors",
         )
     ],
     unit="cursors",
     lineInterpolation="smooth",
-    fillOpacity=10,
     showPoints="auto",
-    valueMin=0,
 )
 
 # Performance Panels
 ops_shard_servers = mongodb_timeseries(
     "Data operations",
     [
         Target(
-            expr=MongoDBMetrics.opcounters("query|getmore"),
+            expr="sum by (job) (rate("
+            + Metrics.SS_OPCOUNTERS('legacy_op_type=~"query|getmore"')
```

Comment:
nit: you can pass a `list` to specify the list of params with a nicer syntax


```suggestion
            + Metrics.SS_OPCOUNTERS(legacy_op_type=['query', 'getmore'])
```